src/lib/viewers/BaseViewer.js

```diff
@@ -997,6 +997,7 @@ class BaseViewer extends EventEmitter {
             initialMode: this.getInitialAnnotationMode(),
             intl: (options && options.intl) || intlUtil.createAnnotatorIntl(),
             modeButtons: ANNOTATION_BUTTONS,
+            stroke: { color: this.annotationModule.getColor() },
```

Comment:
Should this be `initialStroke`, similar to `initialMode`? Do we need mode-specific options to avoid blowing out this top level object?

Reply:
I renamed it to `initialColor`. And this is not mode-specific. The other mode can change color too.